seqr/views/apis/anvil_workspace_api_tests.py

```diff
@@ -183,27 +190,35 @@ def test_create_project_from_workspace(self, mock_file_iter, mock_file_exist, mo
             ['38', 'A test project', TEST_WORKSPACE_NAMESPACE, TEST_NO_PROJECT_WORKSPACE_NAME])
         mock_add_service_account.assert_called_with(self.manager_user, TEST_WORKSPACE_NAMESPACE, TEST_NO_PROJECT_WORKSPACE_NAME)
         mock_has_service_account.assert_not_called()
-        mock_sleep.assert_not_called()
-        mock_file_exist.assert_called_with('gs://test_bucket/test_path.vcf', user=self.manager_user)
-        mock_file_iter.assert_called_with('gs://test_bucket/test_path.vcf', byte_range=None)
-
-        email_body = """
-        test_user_manager@test.com requested to load WES data (GRCh38) from AnVIL workspace "{namespace}/{name}" at 
-        "gs://test_bucket/test_path.vcf" to seqr project {{project_name}} (guid: {guid})
-
-        The sample IDs to load are attached.    
-        """.format(namespace=TEST_WORKSPACE_NAMESPACE, name=TEST_NO_PROJECT_WORKSPACE_NAME, guid=project.guid)
-        mock_email.assert_called_with(
-            subject='AnVIL data loading request',
-            body=email_body.format(project_name=TEST_NO_PROJECT_WORKSPACE_NAME),
-            to=['test_data_manager@test.com', 'test_superuser@test.com'],
-            attachments=[('{}_sample_ids.tsv'.format(project.guid), 'NA19675\nNA19678\nHG00735')]
+        mock_time.sleep.assert_not_called()
+        mock_file_exist.assert_called_with(TEST_VCF_PATH, user=self.manager_user)
+        mock_file_iter.assert_called_with(TEST_VCF_PATH, byte_range=None)
+        mock_tempfile.NamedTemporaryFile.assert_called_with(mode='wb', delete=False)
```

Comment:
might be a bit cleaner if you mock `tempfile.NamedTemporaryFile` instead of just `tempfile`